src/components/SnackChat/SnackChat.js

```diff
@@ -48,24 +56,101 @@ class SnackChat extends Component {
   canvas = React.createRef();
 
   state = {
-    counter: 5,
+    gettingInPosition: true,
+    counter: COUNTDOWN_TIME + LOADING_ANIMATION_TIME + PHOTO_ANIMATION_TIME,
     captured: false
   };
 
   componentDidMount() {
     posenet.load(0.5).then(net => (this.net = net));
-    this.ctx = this.canvas.current.getContext('2d');
-    this.ctx.lineWidth = 5;
-    this.ctx.strokeStyle = 'red';
-    requestAnimationFrame(this.update);
-    this.filter = new Image();
-    this.filter.src = this.props.storeList[this.props.prediction.id].image;
   }
 
   componentWillUnmount() {
     clearInterval(this.timer);
   }
 
+  countdown = async () => {
+    this.timer = setInterval(
+      () => this.setState(prevState => ({counter: prevState.counter - 1})),
+      1500
+    );
+  };
+
+  playGettingInPositionAnimation = () => {
+    const numberOfFallingSnacks = 3;
+    const itemPositions = [];
+    for (let i = 1; i <= numberOfFallingSnacks; i++) {
+      itemPositions.push({
+        x: (this.canvas.current.width * i) / (numberOfFallingSnacks + 1),
+        y:
+          Math.random() * 1.4 * this.canvas.current.height -
```

Comment:
we dont know what 1.4 and 0.2 actually are so perhaps we can have them as names variables